_posts/2020-06-11-Solidity-0610-release-announcement.md

```diff
@@ -58,10 +58,9 @@ A notable new feature is the introduction of error codes. In Standard JSON input
 (``--standard-json``) the error codes will always be available under the property ``"errorCode"``.
 In the command-line interface, you can enable error codes via the switch ``--error-codes``.
 
-Two errors that contain a different error message (e.g. if the error concerns two
-different types) but are still essentially about the same thing will use the same error code.
+Similar errors (for example if error messages differ in type names only) might share the same error code. Vice versa, the same error message reported under different circumstances might have different error codes (although that is not a typical case).
```

Comment:
I think this is a bit too technical.

Reply:
I think it sounds a bit more elegant than the wording we had before.

Reply:
I would at least drop the "might" (if they do not share the same error code, they are just not similar enough...) and if two errors have a different error code then they are just two different errors that differ by error code....

Reply:
Ok then we can go straight back to your wording imo. ;)

Reply:
Tried to use your wording, but slightly rephrased it. Can you check @chriseth?